vllm/envs.py

```diff
@@ -289,6 +290,13 @@ def get_vllm_port() -> Optional[int]:
     lambda: (os.environ.get("VLLM_USE_TRITON_FLASH_ATTN", "True").lower() in
              ("true", "1")),
 
+    # Use separate prefill and decode kernels for V1 attention instead of
+    # the unified triton kernel.
+    "VLLM_V1_USE_PREFILL_DECODE_ATTENTION":
```

Comment:
Nit: `VLLM_V1_TRITON_ATTN_FORCE_PREFILL_DECODE` sounds slightly more accurate to me, but feel free to use the name that works best in your eyes